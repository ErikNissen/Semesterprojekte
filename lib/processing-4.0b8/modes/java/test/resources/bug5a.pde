version https://git-lfs.github.com/spec/v1
oid sha256:cb7e86635900d473ddd9bdce098c2c1b391ab35994402c4d43f2dac683df56be
size 76
