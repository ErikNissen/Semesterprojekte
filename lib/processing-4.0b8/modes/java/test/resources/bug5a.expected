version https://git-lfs.github.com/spec/v1
oid sha256:b7293b76f0ec4ba7be86672112f59a0a9de5fded58959802da03c4ad30e7fd60
size 766
